src/Console/Command/ProcessExportJobs.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * Ushahidi Data Provider Console Commands
+ *
+ * @author    Ushahidi Team <team@ushahidi.com>
+ * @package   Ushahidi\Console
+ * @copyright 2018 Ushahidi
+ * @license   https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+
+namespace Ushahidi\Console\Command;
+
+use Ushahidi\Core\Usecase;
+use Ushahidi\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Ushahidi\Core\Entity\ExportJobRepository;
+
+/* Simple console command that processes pending jobs in the DB */
+
+class ProcessExportJobs extends Command
+{
+
+    private $data;
+    private $exportJobRepository;
+
+
+    public function setExportJobRepo(ExportJobRepository $repo)
+    {
+        $this->exportJobRepository = $repo;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('processexports')
+            ->setDescription('Processes pending export jobs.');
+    }
+
+    protected function getJobs()
+    {
+        // @TODO: handle input options, e.g., to find just pending jobs
+        $jobs = $this->exportJobRepository->getAllJobs();
+        return $jobs;
+    }
+
+    protected function updateJobWithResponse($jobId, $responseInfo)
+    {
+        // update that job with success/failure
+        $resultStatus = 'pending';
+        if (array_key_exists('success', $responseInfo)) {
+            if ($responseInfo['success'] == true) {
+                $resultStatus = 'SUCCESS';
+            } else {
+                $resultStatus = 'FAILED';
+            }
+        }
+
+        $jobEntity = $this->exportJobRepository->get($jobId);
+        // @TODO: get accessible path for this URL from config!
+         $accessiblePath = 'media/uploads/';
```

Comment:
@rjmackay - Is there a recommended path for storing completed files, such that the client can easily access them. If we're not using Kohana, and this isn't running from a webserver, is there a best way to build this path?

Reply:
In Lumen we use the laravel filesystem https://laravel.com/docs/5.6/filesystem or Flysystem directly. Does `Ushahidi\Core\Tool\Uploader::upload()` do what you need here?
If not you might have to use a temporary workaround, then refactor for lumen